Pipfile

```diff
@@ -6,6 +6,7 @@ name = "pypi"
 [packages]
 bottle = "*"
 pip = "==18.0"
+pre-commit = "*"
```

Comment:
Black requirement is listed in `pre-commit-config.yaml` .When i went through the documentation on black, it said to run `pre-commit install` after writing the requirement in the file listed above. I did the same. And black works before commits. I read about pre-commits too, so I think it is right as before commiting, the sent request for webhooks uses the configuration file. There was no installation for pipenv afterwards. So according to me, this is fine.

Reply:
For confirmation, I ran dummy python commits on PyJudge and another repo with python code, PyJudge still executed black and the other one did not.

Reply:
Added using
`pipenv install black`
Locking: dependencies not resolved properly, so used
`pipenv lock --pre`

Reply:
a good way to see where your command is coming from is `which <command>`. I don't know how it might work on other OSes but on linux this tells you if the `black` you are using is from the virtualenv or the global env.